ionoscloud/data_source_server.go

```diff
@@ -366,6 +366,16 @@ func setServerData(d *schema.ResourceData, server *ionoscloud.Server, token *ion
 				entry["ips"] = ips
 			}
 
+			if nic.Properties.Ipv6Ips != nil {
+				var ipv6Ips []interface{}
+				for _, ip := range *nic.Properties.Ipv6Ips {
+					ipv6Ips = append(ipv6Ips, ip)
+				}
+				entry["ipv6_ips"] = ipv6Ips
+			}
+
+			entry["ipv6_cidr_block"] = mongo.ToValueDefault(nic.Properties.Ipv6CidrBlock)
```

Comment:
This uses mongo.ToValueDefault because when this was written there was no ionoscloud.ToValueDefault. But now there is, so better to repace everywhere in this file with ionoscloud.